crates/core/src/catalog.rs

```diff
@@ -6,7 +6,8 @@ use chrono::prelude::*;
 use isahc::AsyncReadResponseExt;
 use serde::{Deserialize, Serialize};
 
-const CATALOG_URL: &str = "https://raw.githubusercontent.com/ajour/catalog/main/catalog-0.1.0.json";
+const CATALOG_URL: &str =
+    "https://raw.githubusercontent.com/ajour/catalog/feat/hub-source/catalog-0.2.0.json";
```

Comment:
Once we are ready to merge this PR, we have to merge: https://github.com/ajour/catalog/pull/2